### ISSUE TITLE
Color contrast of 1.91:1 for expand/collapse on "This version is outdated!"

### ISSUE THREAD

**Issue by goodwitch** [w3c/wcag#312]
Dear W3C,  the black (#000000) "expand" and "collapse" text/feature on dark red (#800000) does not meet color contrast requirements.  This "expand" and "collapse" text/feature is on the red warning that the version of a spec you are looking at is not the latest.  

aXe reports: 
Element has insufficient color contrast of 1.91 (foreground color: #000000, background color: #800000, font size: 12.0pt, font weight: normal). Expected contrast ratio of 4.5:1

**Comment by fstrr** [comment]
Closing this issue due to inactivity. If it needs to be re-opened, please do so (preferably with a link to the relevant content).

**Comment by patrickhlauke** [comment]
assume it referred to the expand/collapse on the dialog warning when going to an older version of the spec, e.g. https://www.w3.org/TR/2008/PR-WCAG20-20081103/

![screenshot of the 'This version is outdated!' dialog](https://user-images.githubusercontent.com/895831/165994201-9f5e9c16-3e59-4883-b00c-77b70cf39081.png)

Looks like the styles have been changed - the text is now white on dark red
